### PR TITLE
Fix: Pause action

### PR DESCRIPTION
**Pull request by erebe**
+ Replicas is not specified in charts so after the pause
    Helm install does not scale back statefulset to 1
  + Force replicas in chart to 1 to ensure that helm upgrade scale it back
  + Fix for MySQL were labels are not applied

https://qovery.atlassian.net/browse/DEV-927

### CODE REVIEW

File: lib/helm-freeze.yaml [hunk 1]
```diff
@@ -44,10 +44,12 @@ charts:
     repo_name: loki
     version: 0.24.0
     comment: https://github.com/grafana/loki/tree/master/production/helm
+    # Chart is frozen due to custom modification see https://github.com/Qovery/engine/pull/293
```
Comment (deimosfr) [inline lib/helm-freeze.yaml:47]: please add no_sync: true to freeze them

File: lib/common/services/redis/templates/redis-master-statefulset.yaml [hunk 2]
```diff
@@ -19,6 +19,7 @@ spec:
       release: {{ .Release.Name }}
       role: master
   serviceName: {{ template "redis.fullname" . }}-headless
+  replicas: 1
```
Comment (deimosfr) [inline lib/common/services/redis/templates/redis-master-statefulset.yaml:22]: why don't you use a variable there?

File: lib/common/services/mongodb/templates/deployment-standalone.yaml [hunk 3]
```diff
@@ -19,6 +19,7 @@ metadata:
 spec:
   {{- if .Values.useStatefulSet }}
   serviceName: {{ template "mongodb.serviceName" . }}
+  replicas: 1
```
Comment (deimosfr) [inline lib/common/services/mongodb/templates/deployment-standalone.yaml:22]: why don't you use a variable there?

File: lib/common/services/redis/templates/redis-master-statefulset.yaml (hunk 2 above)
Comment (erebe) [reply lib/common/services/redis/templates/redis-master-statefulset.yaml:22]: why so ? you can't have more than one in any case. That's why they didn't bother to specify it in the first place

File: lib/common/services/mongodb/templates/deployment-standalone.yaml (hunk 3 above)
Comment (erebe) [reply lib/common/services/mongodb/templates/deployment-standalone.yaml:22]: Same than above

File: lib/helm-freeze.yaml (hunk 1 above)
Comment (erebe) [reply lib/helm-freeze.yaml:47]: added

**Review by deimosfr — APPROVED** [review]

**Pull request merged**